spec/facets/ErrorMessageRunFacet.json

```diff
@@ -0,0 +1,38 @@
+{
+  "$schema" : "https://json-schema.org/draft/2020-12/schema",
+  "$id" : "https://openlineage.io/spec/facets/1-0-0/ErrorMessageRunFacet.json",
+  "$defs" : {
+    "ErrorMessageRunFacet" : {
+      "allOf" : [ {
+        "$ref" : "https://openlineage.io/spec/1-0-2/OpenLineage.json#/$defs/RunFacet"
+      }, {
+        "type" : "object",
+        "properties" : {
+          "message" : {
+            "description" : "A human-readable string representing error message generated by observed system",
+            "type" : "string",
+            "example" : "org.apache.spark.sql.AnalysisException: Table or view not found: wrong_table_name; line 1 pos 14"
+          },
+          "programmingLanguage" : {
+            "description" : "Programming language the observed system uses.",
+            "type" : "string",
+            "example" : "JAVA"
+          },
+          "stackTrace" : {
```

Comment:
Are we able to limit the size of `stakTrace` directly within a Java client to make sure this does not exceed X characters?

Reply:
As with all facets, client itself does not provide that functionality as facets code is autogenerated. We can add it on integration side.